go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/keboola/keboola-as-code
 
 go 1.17
 
-replace github.com/google/go-jsonnet v0.18.0 => github.com/keboola/go-jsonnet v0.18.1-0.20220419142634-2c65675b771a
+replace github.com/google/go-jsonnet v0.18.0 => github.com/keboola/go-jsonnet v0.18.1-0.20220420163252-c8b00f8764e8
```

Comment:
TODO: aktualizovat po mergnuti https://github.com/keboola/go-jsonnet/pull/5 (teraz to ukazuje na branch z PR)

Reply:
:heavy_check_mark: